run_test.sh

```diff
@@ -49,12 +49,14 @@ ln -s ${tmp_dir} test_dir
 cd test_dir
 echo "cd $PWD"
 
-# Clone PICSAR and AMReX
+# Clone PICSAR, AMReX and warpx-data
 git clone https://github.com/AMReX-Codes/amrex.git
 cd amrex && git checkout 87e01fe44b483adab034072dc000aed67aa1b5a7 && cd -
 # Use QED brach for QED tests
 git clone https://github.com/ECP-WarpX/picsar.git
 cd picsar && git checkout c16b642e3dcf860480dd1dd21cefa3874f395773 && cd -
+# warpx-data contains various required data sets
+git clone https://github.com/ECP-WarpX/warpx-data.git
```

Comment:
We can try to speed this up via a shallow clone:
```suggestion
git clone --depth 1 https://github.com/ECP-WarpX/warpx-data.git
```